src/main.py

```diff
@@ -1,8 +1,38 @@
 from fastapi import FastAPI
 
+from fastapi.responses import JSONResponse
+
+from src.auth.utils import login_user, create_user, get_token
+from src.auth.models import User
+
 app = FastAPI()
 
 
-@app.get("/")
-async def root():
-    return {"message": "Hello World"}
+@app.get('/registrate/{username}/{password}/{name}/{surname}')
```

Comment:
It should be post request.
We do not want to provide information like password or names in url